com.unity.render-pipelines.core/Runtime/Common/CoreUnsafeUtils.cs

```diff
@@ -150,6 +151,233 @@ public static void CopyTo<T>(this T[] list, void* dest, int count)
                 UnsafeUtility.WriteArrayElement<T>(dest, i, list[i]);
         }
 
+        private static void RadixCalculateRadixParams(int radixBits, out int bitStates)
+        {
+            bitStates = 1 << radixBits;
+            if (radixBits != 2 && radixBits != 4 && radixBits != 8)
```

Comment:
Perhaps it is just me but I'm used to check before using. What if it is negative or even NaN then ?

Reply:
Done! renamed all these to follow the convention you suggested.